website/README.md

```diff
@@ -2,32 +2,4 @@
 
 This directory contains the portions of [the OpenTofu website](https://opentofu.org) that pertain to the core functionality, excluding providers and the overall configuration.
 
-## Suggesting Changes
-
-You can [submit an issue](https://github.com/opentofu/opentofu/issues/new/choose) with documentation requests or submit a pull request with suggested changes.
-
-Click **Edit this page** at the bottom of any OpenTofu website page to go directly to the associated markdown file in GitHub.
-
-## Modifying Sidebar Navigation
-
-You must update the sidebar navigation when you add or delete documentation .mdx files. If you do not update the navigation, the website deploy preview fails.
-
-To update the sidebar navigation, you must edit the appropriate `nav-data.json` file. This repository contains the sidebar navigation files for the following documentation sets:
-
-- OpenTofu Language: [`language-nav-data.json`](https://github.com/opentofu/opentofu/blob/main/website/data/language-nav-data.json)
-- OpenTofu CLI: [`cli-nav-data.json`](https://github.com/opentofu/opentofu/blob/main/website/data/cli-nav-data.json)
-- Introduction to OpenTofu: [`intro-nav-data.json`](https://github.com/opentofu/opentofu/blob/main/website/data/intro-nav-data.json)
```

Comment:
I think these 2 sections are still valid here, especially the sidebar navigation.

Reply:
How are these used/updated?

Reply:
Okay, so re-reading this, the **edit this page** section is not valid, so we can maybe drop the part about
> Click **Edit this page** at the bottom of any OpenTofu website page to go directly to the associated markdown file in GitHub.

But the sidebar navigation is consumed in the opentofu.org repo and used to create our navigation. 

They're exported here: https://github.com/opentofu/opentofu.org/blob/main/sidebars.js#L1-L26

And we have that being consumed here: https://github.com/opentofu/opentofu.org/blob/main/docusaurus.config.js#L102

Reply:
Oh, thanks, let me change the readme then.